index.html

```diff
@@ -0,0 +1,176 @@
+<!DOCTYPE html>
+<html lang="en">
+	<head>
+		<meta charset="UTF-8" />
+		<meta name="viewport" content="width=device-width, initial-scale=1.0" />
+		<!-- <link rel="stylesheet" href="./dist/index.css" /> -->
+		<link rel="stylesheet" href="/scss/dist/index.css" />
+		<link rel="preconnect" href="https://fonts.googleapis.com" />
+		<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+		<link
+			href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&family=Poppins:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
+			rel="stylesheet"
+		/>
+		<link
+			rel="stylesheet"
+			href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@20..48,100..700,0..1,-50..200"
+		/>
+		<title>Document</title>
+	</head>
+	<body>
+		<header id="header">
+			<div class="top-banner">
+				<div class="banner">
+					<img src="./images/Facebook.svg" alt="" />
+					<img src="./images/twitter.svg" alt="" />
+					<p>Logout</p>
+					<p>My Page</p>
+					<p>English</p>
+				</div>
+			</div>
+			<div class="menu" id="mobile-menu">
+				<div class="menu-bottom">
+					<img src="./images/logo.svg" alt="" />
+					<div class="left-menu">
+						<ul class="left">
+							<li><a class="active" href="#">Home</a></li>
+							<li><a href="./about.html">About</a></li>
+							<li><a href="">Program</a></li>
+							<li><a href="">News</a></li>
+						</ul>
+						<button>CC compaign</button>
+					</div>
+				</div>
+			</div>
+			<div id="humburger">
+				<span class="material-symbols-outlined"> menu </span>
+			</div>
```

Comment:
- [ ] Following the beast practices for using semantic tags, I'll suggest you wrap this in a `nav` tag as this is the main website navigation. :+1: :+1: